main.js

```diff
@@ -0,0 +1,54 @@
+let board = [[5, 4, 3, 2, 1],
+[],
+[]]; 
+
+// use map to create horizontal board view
+
+let viewBoard = (board) => {
+  let boardView = board.map(row => {
+    return '--- ' + row.join(' ')
+  }).join('\n\n');
+  console.log(boardView);
+};
+
+// create function called moveDisk that takes two arguments
+const moveDisk = (currPeg, nextPeg) => {
+  let diskIndex = board[currPeg].pop();
```

Comment:
instead of taking the value off the array, you should check if you need to first so you don't have to push it back - this is creating unnecessary logic